src/main/resources/config/liquibase/master.xml

```diff
@@ -2,7 +2,16 @@
 <databaseChangeLog
     xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
```

Comment:
same, it should not change, because it's already deployed in prod

Reply:
Are you sure about it? I think an update to XML schema location will not impact existing migrations.

Reply:
not sure at all, I never try it. But I'm trusting you, if you say there will be no problem